build.gradle

```diff
@@ -132,9 +132,9 @@ dependencies {
 
     compile 'org.eclipse.jetty:jetty-servlets:9.2.14.v20151106'
 
-    providedCompile 'org.springframework.boot:spring-boot-starter-tomcat:1.3.5.RELEASE'
+    compile 'org.springframework.boot:spring-boot-starter-tomcat:1.3.5.RELEASE'
```

Comment:
If it is not set as _provided_, the dependency will be added to the final distribution, even if not needed. This will increase the final distribution size and may create conflicts due to transitive dependencies that are fetched.